snapcraft/internal/project_loader/errors.py

```diff
@@ -119,3 +121,17 @@ class SnapcraftAfterPartMissingError(ProjectLoaderError):
 
     def __init__(self, part_name, after_part_name):
         super().__init__(part_name=part_name, after_part_name=after_part_name)
+
+
+class SnapcraftProjectUnusedKeyAssetError(snapcraft.internal.errors.SnapcraftException):
+    def __init__(self, key_path: pathlib.Path):
+        self.key_path = key_path
+
+    def get_brief(self) -> str:
+        return f"Found unused key asset {self.key_path!s}."
```

Comment:
self.key_path is of type Path, which probably prevented you from using repr, which led to this that does not add quotes to the string. You might need format here or enclose self.key_path in single quotes to keep the output consistent.

Reply:
Fixed, thnx